packages/fxa-auth-server/lib/senders/templates/partials/paymentProvider.html

```diff
@@ -0,0 +1,10 @@
+<br>
+{{#ifEquals payment_provider "paypal"}}
+  {{t "PayPal" }}
+{{/ifEquals}}
+
+{{#ifEquals payment_provider "stripe"}}
+  {{#if lastFour}}
+    {{t "%(cardType)s card ending in %(lastFour)s" }}
+  {{/if}}
+{{/ifEquals}}
```

Comment:
Would be nice if we can keep in the logic in TS/JS and just pass a string to teh templates.

Reply:
Yeah I agree - especially when there are more than 2 payment providers but I added the `{{#ifEquals}}...{{/ifEquals}}` checks in the templates since that is where the `{{#if lastFour}}..{{/if}}` was/is.